src/bicep-registry/scheduled-action/main.bicep

```diff
@@ -0,0 +1,138 @@
+// @resourceGroup @subscription @tenant
+
+// See https://learn.microsoft.com/rest/api/cost-management/scheduled-actions/create-or-update
+
+//===| Parameters |============================================================
+
+targetScope = 'subscription'
+
+@description('Name of the scheduled action used in the resource ID.')
+param name string
+
+@description('Indicates the kind of scheduled action.')
+@allowed([
+  'Email'
+  'InsightAlert' // @subscription
```

Comment:
This scope directive indicates that this line should only be here for subscription scope. It's removed for other scopes.